src/Compilers/Lua/Portable/Syntax/Syntax.xml

```diff
@@ -1,1742 +1,1997 @@
+        <Kind Name="TypeBinding" />
+        <Field Name="Type" Type="TypeSyntax">
+            <PropertyComment>
+                <summary>The type of the statement (if any).</summary>
+            </PropertyComment>
+        </Field>
+        <Field Name="QuestionToken" Type="SyntaxToken" Optional="true" Override="true">
+            <PropertyComment>
+                <summary>The question mark at the start of the statement (if any).</summary>
+            </PropertyComment>
+            <Kind Name="QuestionToken" />
+        </Field>
+    </Node>
+
+    <Node Name="TupleTypeSyntax" Base="TypeSyntax">
+        <TypeComment>
+            A
+            <summary>Represents a tuple type binding</summary>
+        </TypeComment>
+        <FactoryComment>
+            <summary>
+                Creates a new <see cref="TupleTypeStnax" /> node.
+            </summary>
+        </FactoryComment>
+        <Kind Name="TypeBinding" />
+        <Field Name="OpenParenthesisToken" Type="SyntaxToken">
+            <PropertyComment>
+                <summary>The parameter list opening parenthesis.</summary>
+            </PropertyComment>
+            <Kind Name="OpenParenthesisToken" />
+        </Field>
+        <Field Name="Types" Type="SeparatedSyntaxList&lt;TypeSyntax&gt;">
+            <PropertyComment>
+                <summary>A list of types.</summary>
+            </PropertyComment>
+        </Field>
+        <Field Name="CloseParenthesisToken" Type="SyntaxToken">
+            <PropertyComment>
+                <summary>The close parenthesis token.</summary>
+            </PropertyComment>
+            <Kind Name="CloseParenthesisToken" />
+        </Field>
+    </Node>
+
+    <Node Name="TableTypeSyntax" Base="TypeSyntax">
+        <TypeComment>
+            <summary>Represents a table type binding</summary>
+        </TypeComment>
+        <FactoryComment>
+            <summary>
+                Creates a new <see cref="TableTypeSyntax" /> node.
+            </summary>
+        </FactoryComment>
+        <Kind Name="TypeBinding" />
+        <Field Name="ColonToken" Type="SyntaxToken" Optional="true" Override="true">
+            <PropertyComment>
+                <summary>The colon at the start of the statement (if any).</summary>
+            </PropertyComment>
+            <Kind Name="ColonToken" />
+        </Field>
+        <Field Name="Types" Type="SeparatedSyntaxList&lt;TableElementTypeSyntax&gt;">
+            <PropertyComment>
+                <summary>A list of types.</summary>
+            </PropertyComment>
+        </Field>
+    </Node>
+
+    <Node Name="VarargTypeSyntax" Base="TypeSyntax">
+        <TypeComment>
+            <summary>Represents a vararg type binding</summary>
+        </TypeComment>
+        <FactoryComment>
+            <summary>
+                Creates a new <see cref="VarargTypeSyntax" /> node.
+            </summary>
+        </FactoryComment>
+        <Kind Name="TypeBinding" />
+    </Node>
```

Comment:
There's no vararg type node, the vararg in function parameters will be added on the `VarargParameter` node itself as a `TypeBindingSyntax`.
```suggestion
```